frappe/utils/boilerplate.py

```diff
@@ -333,6 +333,9 @@ def make_boilerplate(dest, app_name, no_git=False):
 # 	"{app_name}.auth.validate"
 # ]
 
+# Make link fields search translated document names for these DocTypes
+# Recommended only for DocTypes like Role which have limited documents with untranslated names
+# translated_search_doctypes = []
```

Comment:
- Added section for Translation
- This was going over 88 characters, so added examples on a separate line 😅 
- Added newline at EOF.

```suggestion
# Translation
#---------------------------------

# Make link fields search translated document names for these DocTypes
# Recommended only for DocTypes which have limited documents with untranslated names
# For example: Role, Gender, etc.
# translated_search_doctypes = []

```